### ISSUE TITLE
Link to windows precompiled onioncat is dead

### ISSUE THREAD

**Issue by gits7r** [rahra/onioncat#42]
The link @ cypherpunk.at for the Windows precompiled Onioncat is dead. I'd like to mirror a host to it, if you'd permit. Otherwise, we can have it on github as well along with the compilation script and dependencies? Thanks.

**Comment by rahra** [comment]
Yes, unfortunately the domain vanished. It wasn't in my responsibility.
Look here: https://www.abenteuerland.at/ocat/download/
Yes, of course you can mirror it.

**Comment by gits7r** [comment]
thank you very much. by looking at the tags in on `master` it looks like the latest release version is `v0.4.7` while the linked download directory contains up to `v0.38` - is this expected? Also, are there any instructions anywhere how one can compile it into an .exe and host it?

**Comment by rahra** [comment]
There's no special reason except that I simply didn't build a newer release for Windows.
I'll make a new version as soon as possible.
